data_steward/.gcloudignore

```diff
@@ -0,0 +1,95 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
+# using gcloud. It follows the same syntax as .gitignore, with the addition of
+# "#!include" directives (which insert the entries of the given .gitignore-style
+# file at that point).
+#
+# For more information, run:
+#   $ gcloud topic gcloudignore
+#
+.gcloudignore
+.git
+.gitignore
+
+# Duplicating .gitignore entries below because
+# the gcloudignore include directive does not accept paths (i.e. "../.gitignore")
+# and to reduce potential issues with *ignore inconsistencies
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+pip-wheel-metadata/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit test / coverage reports
+.tox/
+.nox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+.pytest_cache/
+
+# Django stuff:
+*.log
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
+
+# Flask stuff:
+instance/
+.webassets-cache
+
+# Jupyter Notebook
+.ipynb_checkpoints
+
+# IPython
+profile_default/
+ipython_config.py
+
+# pyenv
+.python-version
+
+# Environments
+*_env/
```

Comment:
One other suggestion.  You might consider ignoring logging directories (`logs/` or `LOGS/`), rather than just files that end in `.log`.  The deid software saves the SQL it creates in `*.sql` files in the `LOGS` directory.  I don't think we have a conventional logging directory location, but a directory name like `LOGS` is fairly common.  Since deid logs to it, it contains `*.log` and `*.sql` files, none of which are needed by GAE.  It would be nice to start logging all of this stuff (when it runs locally) to a common location.

Everything else looks fine.

Reply:
Good idea @lrwb-aou. We wouldn't want to deploy these files and be misled by log files we may have uploaded.